ansible/roles/operator-pipeline/templates/openshift/tasks/preflight-trigger.yml

```diff
@@ -0,0 +1,116 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preflight-trigger
+spec:
+  params:
+    - default: '4.8'
```

Comment:
I'd probably avoid setting a default here, I think we'll always supply it (see my other comment)

Reply:
Yes we are not going to set the at all and are going to depend on the max_supported_ocp_version as you noted in your previous comment. It is hardcoded for now to ensure we run against a version dptp definitely has in their Hive clusterpool.

Reply:
Indeed. I think we need to land on a list of supported versions _today_, after which @mrhillsman and I can make sure DPTP has the right configurables that preflight-trigger can reach into before we remove this default value.